docs/reference/server-worker-configuration.md

```diff
@@ -14,10 +14,19 @@ The file may be called by any name but should have the following structure:
       "radius": 2
     }
   },
+  "streamingQuery": [
+    ...
+  ],
   "componentDelivery": {
     "default": "RELIABLE_ORDERED",
     "checkoutAllInitially": true
   },
+  "componentSettings": {
+    ...
+  },
+  "permissions": [
```

Comment:
Same as client, don't think permissions is an array

Reply:
See comment above.

Reply:
Fixed.